modules/distributor/distributor.go

```diff
@@ -87,6 +87,11 @@ var (
 		Name:      "distributor_bytes_received_total",
 		Help:      "The total number of proto bytes received per tenant",
 	}, []string{"tenant"})
+	metricIngressBytes = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "tempo",
+		Name:      "distributor_ingress_bytes_total",
+		Help:      "The total number of bytes received per tenant, before limits",
```

Comment:
can we update the help text on `distributor_bytes_received_total` to mention that it's after limits (actual bytes ingested into the system).

also a nit: before limits is bit open, maybe "before rate limits" or "before ingestion limits" would be more clear here